Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/exportSetting.js

```diff
@@ -0,0 +1,71 @@
+angular.module('virtoCommerce.exportModule')
+    .controller('virtoCommerce.exportModule.exporterController', ['$scope', 'platformWebApp.bladeNavigationService', 'virtoCommerce.exportModule.exportModuleApi', function ($scope, bladeNavigationService, exportApi) {
+        var blade = $scope.blade;
+        blade.canStartProcess = false;
+        blade.exportDataRequest = {};
+        blade.isLoading = true;
+
+        function initializeBlade() {
+            blade.exportDataRequest.dataQuery = blade.dataQuery;
+
+            exportApi.getKnowntypes(function (result) {
+                if (result && result.length > 0 && blade.exportDataRequest.dataQuery.exportTypeName) {
```

Comment:
`result.length > 0`
could be just 
`result.length`

Reply:
Is this condition needed here
&& blade.exportDataRequest.dataQuery.exportTypeName
Maybe we should not make a request at all if it is empty, or fill blade.knowntypes and then check it?